src/export/ExportMP3.cpp

```diff
@@ -1859,7 +1858,17 @@ ProgressResult ExportMP3::Export(AudacityProject *project,
    // Verify sample rate
    if (!make_iterator_range( sampRates ).contains( rate ) ||
       (rate < lowrate) || (rate > highrate)) {
-      rate = AskResample(bitrate, rate, lowrate, highrate);
+       if (project->mBatch) {
```

Comment:
Does this compile?  The member is mBatchMode.

That would correct the first version of your fix, for the interactive case, in which behavior should not change.

Reply:
Apologies. I think I might have accidentally changed it before pushing the code. I will fix it.